src/ZendeskApi.Client/Configuration/Headers.cs

```diff
@@ -1,45 +0,0 @@
-﻿using System.Collections;
```

Comment:
`ZendeskClientApi` only has one argument for `CreateClient` which is for the resource. I know that you can change the headers on the resulting `HttpClient` but you never see those clients as a user as they are wrapped by a given resource.